content/en/docs/instrumentation/js/instrumentation.md

```diff
@@ -902,6 +902,75 @@ app.get('/', (_req, _res) => {
 {{< /tab >}}
 {{< /tabpane>}}
 
+### Using Observable (aka Async) Counters
+
+Observable counters can be used to measure an additive, non-negative, monotonically increasing value. They are
+useful when increasing the counter is NOT computationally cheap, the increments happen at very high frequencies, or we simply derive no value from knowing the precise timestamp of increments.
```

Comment:
> They are
useful when increasing the counter is NOT computationally cheap, the increments happen at very high frequencies, or we simply derive no value from knowing the precise timestamp of increments.

if this applies to all async/observable then maybe there should be a top level section calling that out?